### PR TITLE
docs: tegel regression 

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Docs regression check

**Solving issue**  
Checking various small typos, wordings, grammar...

**How to test**  
1. Go commit by commit below to see what is updated.
2. Flag if something is off

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

### CODE REVIEW

File: core/src/components/accordion/accordion-item/accordion-item.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export class TdsAccordionItem {
   /** The header gives users the context about the additional information available inside the panel */
   @Prop() header: string = '';
 
-  /** Changes where the expand icon is placed. */
+  /** Changes position where of expand icon. */
```
Comment (theJohnnyMe) [inline core/src/components/accordion/accordion-item/accordion-item.tsx:12]:
```suggestion
  /** Changes the position of expand icon. */
```

**Review by mJarsater — APPROVED** [review]
Nice! Just one small capitalized thing

File: core/src/components/accordion/accordion-item/accordion-item.tsx (hunk 1 above)
Comment (mJarsater) [inline core/src/components/accordion/accordion-item/accordion-item.tsx:12]:
```suggestion
  /** Changes position of the expand icon. */
```

File: core/src/components/accordion/accordion.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { Component, h, Host, Prop } from '@stencil/core';
   shadow: true,
 })
 export class TdsAccordion {
-  /** Set the variant of the the accordion. */
+  /** Set the variant of the accordion. */
```
Comment (mJarsater) [inline core/src/components/accordion/accordion.tsx:9]:
```suggestion
  /** Set the variant of the Accordion. */
```

**Pull request merged**